src/scales/scale.linear.js

```diff
@@ -121,6 +146,12 @@ module.exports = function(Chart) {
 				});
 			}
 
+			// Take a snapshot of scale limit to be used if invalid values for { max, min }
+			me.takeScaleLimitSnapShot(me.min, me.max);
```

Comment:
It should, i've added the [test case ](https://github.com/chartjs/Chart.js/pull/3792/files#diff-2596fda3dfba9fb9ade16c3abbf09601) for that. This will test should cover that right ?